### PR TITLE
Blazor client-side publish directory

### LINKED ISSUE #10745: Clarify Blazor client-side publish directory location
From @guardrex at [Blazor directory structure topic](https://github.com/aspnet/Blazor.Docs/issues/61) ...

Describe the published output of Blazor client- and server-side apps.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #10745

Let's clarify the `dist` location and not sweat the contents. No one is asking for more detail on what's in the published assets (specific folder and files) at this time.

### CODE REVIEW

File: aspnetcore/host-and-deploy/blazor/client-side.md [hunk 1]
```diff
@@ -184,6 +184,8 @@ For information on deploying to Azure App Service, see <xref:tutorials/publish-t
 
 A *standalone deployment* serves the client-side Blazor app as a set of static files that are requested directly by clients. A web server isn't used to serve the Blazor app.
```
Comment (danroth27) [inline aspnetcore/host-and-deploy/blazor/client-side.md]: > A web server isn't used to serve the Blazor app.

I know this isn't part of this PR, but this statement sounds wrong. You still need a web server to host your Blazor app files.

Comment (guardrex) [reply aspnetcore/host-and-deploy/blazor/client-side.md]: Serverless static file deployment. I should say it's not strictly required. I'll fix it.

Comment (guardrex) [reply aspnetcore/host-and-deploy/blazor/client-side.md]: 🤔 

> Any static file server is able to serve the Blazor app.

**Comment by guardrex** [comment]
@danroth27 Is it ok to merge this one?

**Pull request merged**